index.js

```diff
@@ -194,9 +195,34 @@ var parseSubprojects = function (ctx) {
   var promises = [];
   if (ctx.herman.subprojects) {
     ctx.subprojects = {};
-    Object.keys(ctx.herman.subprojects).forEach(function (name) {
-      var prjCtx = extend({}, ctx.herman.subprojects[name]);
-      var promise = sassdoc.parse(prjCtx.src, prjCtx).then(function (data) {
+    ctx.herman.subprojects.forEach(function (name) {
+      var prjPath = './node_modules/' + name + '/';
+      var configFile = prjPath + '.sassdocrc';
+      var config = {};
+      try {
+        // Load .sassdocrc configuration from subproject directory
+        config = yaml.safeLoad(fs.readFileSync(configFile, 'utf-8'));
+      } catch (err) {
+        ctx.logger.warn('No .sassdocrc found for subproject: ' + name);
```

Comment:
This message is misleading if the problem is malformatted YAML rather than a missing file.